tools/openapi-plugin/src/generators/client/generator.ts

```diff
@@ -113,4 +122,24 @@ async function copySchema(
     }
 }
 
+async function validate(schemaPath: string) {
+    return new Promise((resolve, reject) => {
+        const child = spawn(
+            'npx',
```

Comment:
I thought we agreed to just spawn `npx @redocly/cli lint` here, right?

Reply:
This change was in there :thinking: i mustve removed it accidently.